.circleci/config.yml

```diff
@@ -34,3 +31,14 @@ jobs:
       - run:
           name: Lint and Test packages
           command: yarn test
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8
```

Comment:
IDK if it's necessary to run with node10 and node8. 🤔

Reply:
Node 8 is LTS and 10 current stable release. They are very different and both should be checked.

Reply:
is this going to slow down the test build significantly?
if not, sure why not?

Reply:
It should not as CircleCI runs them in parallel.

Reply:
Would be great to have CircleCI enabled for PRs.